CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ All notable changes to this project will be documented in this file.
 
 ## Added
 
+- Support for `install` command to show user an error if the package(s) content is revoked from 
```

Comment:
Perhaps like

> Support for package revocation added, `criticalup install` will verify packages have not been revoked (due to, for example, a security event) before installation.

We want users to know we are prepared in case something does happen (even though nothing has happened), while also being reaffirming that we're only revoking packages in otherwise unfortunate circumstances. (Not just... for example... because we want to.)

Reply:
Thank you!